PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -102,10 +102,11 @@
 * C extensions suffixed by the platform-specific shared library filetype (e.g.,
   `.so` under Linux, `.dll` under Windows).
 
-The keys of this dictionary are `.`-prefixed filetypes (e.g., `.py`, `.so');
+The keys of this dictionary are `.`-prefixed filetypes (e.g., `.py`, `.so`) or
+`-`-prefixed filetypes (e.g., `-cpython-37m.dll`);
```

Comment:
Interesting, I've never sees this. Can you point me to some example?

Reply:
Search for `-cpython-37m.dll` on https://packages.msys2.org/package/mingw-w64-x86_64-python3?repo=mingw64